tensorflow/jobs/docker/2.14/py3/requirements.txt

```diff
@@ -1,23 +1,24 @@
-amazon-braket-default-simulator==1.26.0
-amazon-braket-pennylane-plugin==1.28.0
-amazon-braket-schemas==1.22.0
-amazon-braket-sdk==1.87.0
-awscli==1.34.22
-botocore==1.35.22
-boto3==1.35.22
+amazon-braket-default-simulator==1.26.1
+amazon-braket-pennylane-plugin==1.30.2
+amazon-braket-schemas==1.22.3
+amazon-braket-sdk==1.88.2.post0
```

Comment:
There's a difference between what we have for examples and what we are using here?

Reply:
I used the dependabot version here, and updated the examples repo manually - that's why they are slightly different. It shouldn't matter here.